### PR TITLE
Correct argument doc for BasicLSTMCell.call

### PR DESCRIPTION
**Pull request by yanboliang**

### CODE REVIEW

File: tensorflow/python/ops/rnn_cell_impl.py [hunk 1]
```diff
@@ -556,9 +556,9 @@ def call(self, inputs, state):
     Args:
       inputs: `2-D` tensor with shape `[batch_size, input_size]`.
       state: An `LSTMStateTuple` of state tensors, each shaped
-        `[batch_size, self.state_size]`, if `state_is_tuple` has been set to
```
Comment (yanboliang) [inline tensorflow/python/ops/rnn_cell_impl.py]: ```self.state_size``` is an ```LSTMStateTuple``` if ```state_is_tuple``` is ```True```, so here should be ```self._num_units```.

Comment (ebrevdo) [reply tensorflow/python/ops/rnn_cell_impl.py]: replace `self._num_units` with just `num_units`.

**Review by ebrevdo — DISMISSED** [review]

File: tensorflow/python/ops/rnn_cell_impl.py (hunk 1 above)
Comment (yanboliang) [reply tensorflow/python/ops/rnn_cell_impl.py]: Done. Thanks.

Comment (yanboliang) [reply tensorflow/python/ops/rnn_cell_impl.py]: @ebrevdo Done. Thanks.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by ekelsen — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ebrevdo** [comment]
Please fix errors.

**Comment by yanboliang** [comment]
@ebrevdo All checks have passed now. Thanks.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request merged**